FAQ.md

```diff
@@ -1,7 +1,143 @@
+
+Nevertheless, third parties such as Google Play app store and the OVHcloud hosting provider might collect some user access information like users’ IP address and user-agent. We have no control of that and seek no access to such information.
+
+Also note that if a file is sent to another device directly, the two parties could learn each others’ IP address.
+
+For more details, see our [privacy policy](https://github.com/LeastAuthority/destiny/blob/main/PRIVACY-POLICY.md). 
+
+**Has Destiny undergone a security audit?**
+
+Destiny has been audited by an independent security audit company, Radically Open Security. You can find their findings and our remediation [here]. 
+
+## **More help:** 
+
+**Why does the app need to stay open for both the sender and the receiver for the transfer to be completed?**
+ 
+Destiny is meant for synchronous file transfers. Files are sent from a sender’s device to a recipient’s device. Unlike some other services, they are not uploaded to and stored on a central server. This means that if the sender is no longer connected, they can no longer offer the file for the receiver to get. 
+
+**How can a simple code like 7-guitarist-revenge be secure? And why can a code only be used once?**
+ 
+There are 65,536 different combinations of the current wordlist used by Destiny. Anyone who tries to guess the right word combination has a 1 in 65,536 chance to guess right, which is a very small chance. If they guess wrong, the transfer is invalidated. This is tough luck for the sender, but also a safety feature: It prevents people from being able to keep guessing until they hit the right word combination. 
+
+This also explains why codes are single use. Once a code is used ‘successfully’ (sender and recipient connect) or ‘unsuccessfully’ (a wrong code combination is guessed), the code is cleared for new usage. If the code would stay ‘alive’ for other recipients, it would lose the security of being hard to guess.
+
+**Can I send multiple files at the same time?**
+  
+If you would like to send multiple files at the same time, you can compress them, for example, as a single ZIP file. But you cannot upload an unzipped folder or select multiple files to upload at once. You will need to send them separately and provide the receiver with a new code each time. 
+
+
+**At what point can I retract a file that I am sending?** 
+
+As a sender you can first cancel a transfer before sharing the code by pressing the Cancel button. After sharing the code you can still press Cancel if the other party has not yet started downloading the file. If you press Cancel while the transfer is ongoing, it is possible for the receiver to already have a portion of the file.
+
+**Can I use Destiny to send/receive files to/from other Magic Wormhole applications?**  
+
+Yes. Make sure to configure the other Magic Wormhole application to use the same environment settings set in Destiny. These are currently:
+
+	Mailbox URL: wss://mailbox.w.leastauthority.com/v1
```

Comment:
We have new address:
Mailbox: wss://mailbox.mw.leastauthority.com/v1
Relay: tcp://relay.mw.leastauthority.com:4001

Reply:
updated